### DISCUSSION TITLE
Failed submission run

### DISCUSSION THREAD

**Discussion by essmehdi** [geeksblabla/blanat#308] [General]
Hello! I noticed that my submission failed to run in the logs. How could that fail with "No such file or directory" knowing that it reads the file correctly from the current directory 😅?

**Comment by aboullaite** [comment]
Thank you for reporting. We'll have a look

**Comment by aboullaite** [comment]
A new logfile has been pushed, please check https://github.com/geeksblabla/blanat/pull/310
